deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -79,10 +79,13 @@ spec:
   graphing:
     enabled: false
     grafana:
-      ingressEnabled: false
+      ingressEnabled: true
```

Comment:
I figured if you go to the trouble of enabling dashboarding you probably want to access it.

Reply:
I think this was our default because we wanted to limit the attack surface but in this case, if you want dashboarding, you're probably fine with exposing the route :)

Reply:
(heh I was curious and ran a git blame, and the defaults/main.yml for this setting has been there since STF 1.1.0, so I don't think there was anything as smart as limiting attack surface... it was probably just the default value that Grafana Operator was using, and we likely just copied it? 🤷🏻 )